.registryrc

```diff
@@ -1 +1 @@
-fad1ad9000a5795ed5c617f6ab677d330f52a3b3
+396ca9e339efcf3ef8a6359a61c0586d23d17d26
```

Comment:
note to update this after merging https://github.com/hyperlane-xyz/hyperlane-registry/pull/817